content/blog/2024/01/2024-01-17-jenkins-contributor-summit-update.adoc

```diff
@@ -0,0 +1,43 @@
+---
+layout: post
+title: "2024 Jenkins Contributor Summit - Update"
+tags:
+- events
+- community
+- contribute
+author: jmMeessen
+discourse: true
+opengraph:
+  image: /images/post-images/2023/11/ContribSummit/Contributor_summit_2024.png
+description: >
+  We are looking forward to meet you during the  Jenkins Contributor Summit in Brussels prior to FOSDEM.
+---
+
+The Jenkins Contributor Summit, being held in Brussels on February 2nd, is now quickly approaching.
+We are looking forward to meeting you.
+
+If you plan to join us and didn’t let us know yet, don’t forget to register by dropping us a note.
+The list of registered attendees can be found in this document: link:https://docs.google.com/spreadsheets/d/1fatDxa39U-yHW6iTpMQZuW_pMf8G9jhO6yczfoqcY08/edit?usp=sharing[Contributor Summit 24 - Attendees].
+
+The Summit will take place at link:https://www.betacowork.com/[**Betacowork**], located at *4 Rue des Pères Blancs, 1040 Brussels, Belgium* (link:https://maps.app.goo.gl/S8VrWsrErmLMXqza7[Betacowork] on Google Maps) in the "*AZZAR*" room.
+
+Find hereafter a quick overview of the day’s planning:
```

Comment:
```suggestion
The following list provides a quick overview of the day’s planning:
```
This suggestion is to better connect this sentence and the following information.